radis/test/tools/test_database.py

```diff
@@ -115,17 +115,43 @@ def test_plot_spec(plot=True, close_plots=True, verbose=True, *args, **kwargs):
         plot_spec(getTestFile("N2C_specair_380nm.spec"))
 
 
-def _run_testcases(plot=True, close_plots=False, verbose=True, *args, **kwargs):
-
-    test_speclist(*args, **kwargs)
-    test_database_functions(
-        plot=plot, close_plots=close_plots, verbose=verbose, *args, **kwargs
+def test_save_compressed2():
```

Comment:
if using this example, Pytest will require an internet connection to download the database. We usually mark these with
```@pytest.mark.needs_connection``` to skip these tests if there is no connection. 

However, here you could calculate a spectrum with the Test Line databases directly. The code below would work too, and save the connection:
```
from radis.test.utils import setup_test_line_databases
from radis import calc_spectrum

setup_test_line_databases()
s = calc_spectrum(
    2000,
    2300,  # cm-1
    molecule="CO",
    isotope="1,2,3",
    pressure=1.01325,  # bar
    Tgas=700,  # K
    mole_fraction=0.1,
    path_length=1,  # cm
    verbose=False,
    wstep=0.01,
    medium="vacuum",
    databank="HITRAN-CO-TEST")
```